pyxlma/lmalib/flash/properties.py

```diff
@@ -123,7 +123,8 @@ def event_discharge_energy(z,area):
     eta_c      = 0.004 #Scale the energy to depict the fraction of energy neutralized by
                        #each flash in the capacitor model
     #Capacitor model:
-    w = 4 * ((sigma_crit**2. * d * area)/(2* e)) #The quantity for appears when considering image charges (2*sigma)^2=4sigma^2
+    w = 4 * ((sigma_crit**2. * d * area.iloc[0])/(2* e)) #The quantity for appears when considering image charges (2*sigma)^2=4sigma^2
+    print(f'area={area.iloc[0]}')
```

Comment:
Please remove print statement.

Reply:
Whoops, I forgot I left that in there. Thanks.